packages/core/src/templates/reactTypescriptTemplate.js

```diff
@@ -0,0 +1,34 @@
+export default (code, opts = {}, state) => {
+  let props = ''
+
+  if (opts.expandProps && opts.ref && opts.titleProp) {
+    props = '{svgRef: string, title: string, ...props}'
+  } else if (opts.expandProps && opts.titleProp) {
+    props = '{title: string, ...props}'
+  } else if (opts.expandProps && opts.ref) {
+    props = '{svgRef: string, ...props}'
+  } else if (opts.titleProp && opts.ref) {
+    props = '{svgRef: string, title: string}'
+  } else if (opts.expandProps) {
+    props = 'props: any'
```

Comment:
I've had great success here, typing the props as `SVGAttributes<SVGElement>` imported with `import React, { SVGAttributes } from 'react';` instead of `any` type.

Mind though, that you would then need to create a few union types for the cases above, such as `{svgRef, ...props}: SVGProps<SVGSVGElement> & {svgRef: string}`